draft-ietf-httpbis-resumable-upload.md

```diff
@@ -777,6 +792,17 @@ Recommended HTTP status code:
 Reference:
 : This document
 
+IANA is asked to register the following entry in the "HTTP Problem Types" registry:
+
+Type URI:
+: https://iana.org/assignments/http-problem-types#inconsistent-upload-length
+Title:
+: Inconsistent Upload Length Values
+Recommended HTTP status code:
+: 400
```

Comment:
409?

Reply:
I used 400 for now to save the use of 409 to mismatching offsets in this document. Depending on how the discussion following from https://github.com/httpwg/http-extensions/pull/3018 turns out, we might get a different status code for mismatching offsets.

Reply:
The example above shows 400 though

Reply:
Oh my bad! Thank you, I'll fix that!